packages/payment-processor/test/payment/encoder-approval.test.ts

```diff
@@ -55,6 +55,7 @@ const approvalSettings = {
 const mnemonic = 'candy maple cake sugar pudding cream honey rich smooth crumble sweet treat';
```

Comment:
There's a bit of duplication in the payment-processor tests file, but I didn't want to refactor them in this PR